javascript-forms/form-values.js

```diff
@@ -0,0 +1,12 @@
+var messageData = {};
```

Comment:
You should move this into the `handleContactForm` function so it doesn't continue to exist after the function ends.

Reply:
Copy that!